backend/repository/handler_test.go

```diff
@@ -68,7 +68,7 @@ func TestHandlers(t *testing.T) {
 		for _, tc := range testCases {
 			tc := tc // Capture range variable
 			t.Run(tc.Name, func(t *testing.T) {
-				t.Parallel()
+				//t.Parallel()
```

Comment:
i think reading the middleware config files or doing schema validation concurrently;  breaks parallelism. It just hangs for some reason.  open to suggestions.   This runs quickly even in serial  so I think it might be an optimization we can leave out for the the time being

Reply:
dont need parallel in tests for now but there might be race conditions since we handle http requests in parallel.

We should look into this.